examples/rn-airbnb/README.md

```diff
@@ -0,0 +1,143 @@
+* `OpenRealmBehaviorType.OpenImmediately`
+  * Opens the Realm file immediately if it exists, otherwise it first creates a new empty Realm file then opens it.
+  * This lets users use the app with the existing data, while syncing any changes to the device in the background.
+* `OpenRealmBehaviorType.DownloadBeforeOpen`
+  * If there is data to be downloaded, this waits for the data to be fully synced before opening the Realm.
+
+This app opens a Realm via `RealmProvider` (see [AuthenticatedApp.tsx](./frontend/app/AuthenticatedApp.tsx)) and passes the configuration as props. We use `OpenImmediately` for new and existing Realm files in order to use the app while offline.
+
+> See [OpenRealmBehaviorConfiguration](https://www.mongodb.com/docs/realm-sdks/js/latest/types/OpenRealmBehaviorConfiguration.html) for possible configurations of new and existing Realm file behaviors.
+
+## Getting Started
+
+### Prerequisites
+
+* [Node.js](https://nodejs.org/)
+* [React Native development environment](https://reactnative.dev/docs/environment-setup?guide=native)
+  * Refer to the **"React Native CLI Quickstart"**.
+
+### Set up an Atlas Database with a Sample Dataset
+
+1. [Deploy a free Atlas cluster](https://www.mongodb.com/docs/atlas/getting-started/#get-started-with-atlas) and create an Atlas database.
+2. [Load the Sample Airbnb Dataset](https://www.mongodb.com/docs/atlas/sample-data/) into your Atlas database.
+    * Several databases and collections exist in the sample dataset, but we will only be using the `sample_airbnb` database and its `listingsAndReviews` collection.
+3. [Create a Search Index](https://www.mongodb.com/docs/atlas/atlas-search/tutorial/create-index/) with an Index Name of `airbnb`.  This will be used for Atlas Search within the application
+
+### Set up an Atlas App Services App via CLI
+
+To import and deploy changes from your local directory to App Services you can use the command line interface:
+
+1. [Set up App Services CLI](https://www.mongodb.com/docs/atlas/app-services/cli/).
+2. In the provided [backend directory](./backend/) (the App Services App), update the following:
+    * Cluster Name
+      * Update the `"clusterName"` in [data_sources/mongodb-atlas/config.json](./backend/data_sources/mongodb-atlas/config.json) to the name of your cluster.
+      * (The default name is `Cluster0`.)
+    * App ID
+      * There is no `"app_id"` defined in [realm_config.json](./backend/realm_config.json) since we will create a brand new App. **If** you for some reason are updating an existing app, add an `"app_id"` field and its value.
+3. [Create and deploy](https://www.mongodb.com/docs/atlas/app-services/cli/appservices-apps-create/) the local directory to App Services:
+```sh
+appservices push --local ./backend
```

Comment:
I tried running this and I get
```
push setup failed: directory './backend' is not a supported App Services project
```
Might be some mess up on my end though. Running `appservices version 1.5.0`

Reply:
I'm getting it as well.  I'll verify what's missing.  the CLI is not super intuitive

Reply:
Should work now

Reply:
Hmm, it still doesn't for me

Reply:
mmm...the .mdb folder didn't get added.